src/ResourceManager/KeyVault/Commands.KeyVault/Models/KeyAttributes.cs

```diff
@@ -65,7 +65,7 @@ public KeyAttributes()
 
         public DateTime? Updated { get; private set; }
 
-        public bool PurgeDisabled { get; private set; }
+        public string RecoveryLevel { get; private set; }
```

Comment:
We don't have the option of the client being broken, we will not ship any breaking changes until November, except in the case of critical security issues or other critical bugs.